.github/workflows/go-storage-auto-build.yml

```diff
@@ -0,0 +1,37 @@
+name: Dependabot go-storage auto build
+on: pull_request_target
+
+permissions:
+  pull-requests: write
+  contents: write
```

Comment:
```suggestion
on: pull_request
```
is enough?

Reply:
We need `Auto commit & push`, so `write` is required.